src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure EventGrid Library")]
 [assembly: AssemblyDescription("Provides developers with a library to publish events to Azure EventGrid and to consume events delivered by EventGrid.")]
 
-[assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("4.0.0.0")]
```

Comment:
The change in this file should be reverted as this is the dataplane assemblyinfo.

Reply:
fix